tools/emcoverage.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import errno
+import os
+import shutil
+import sys
+import uuid
+from glob import glob
+
+from coverage.cmdline import main
+
+if __name__ == '__main__':
+  sys.executable = os.path.abspath(__file__)
```

Comment:
Please comment this insanity.

Reply:
Oh man.. that is kind of a crazy (an clever) hack.   How about we find another way to set the python used by emscripten.   Perhaps via the PYTHON environment variable?

Reply:
Yes, `EM_PYTHON` is probably more consistent than relying on hacking `sys.executable`.